tests/test_data_stores.py

```diff
@@ -0,0 +1,341 @@
+from datetime import datetime, timezone
+from unittest.mock import MagicMock
+
+import numpy as np
+import pandas as pd
+import pytest
+from cognite.client.exceptions import CogniteAPIError
+
+from cognite_synthetic_tags import data_stores
+
+
+@pytest.fixture
+def mocked_client():
+    p_client = MagicMock()
+    p_retrieve = p_client.datapoints.retrieve.return_value
+    p_retrieve.to_pandas.return_value = pd.DataFrame(
+        columns=["houston.ro.REMOTE_AI[22]"],
+        data=[
+            [0.003925000131130218],
+            [np.nan],
+            [np.nan],
+        ],
+        index=map(
+            lambda val: datetime.strptime(val, "%Y-%m-%dT%H:%M:%S"),
+            [
+                "2020-01-01T00:00:53",
+                "2020-01-01T00:00:54",
+                "2020-01-01T00:00:55",
+            ],
+        ),
+    )
```

Comment:
More code golfing (can be applied to several other occurrences below):
```suggestion
    p_retrieve.to_pandas.return_value = pd.DataFrame(
        {"houston.ro.REMOTE_AI[22]": [0.003925000131130218, np.nan, np.nan]},
        index=[
            "2020-01-01T00:00:53",
            "2020-01-01T00:00:54",
            "2020-01-01T00:00:55",
        ],
    )
```